target/linux/ath79/dts/qca9533_mikrotik_routerboard-lhg-2nd.dts

```diff
@@ -0,0 +1,175 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca953x.dtsi"
+
+/ {
+	compatible = "mikrotik,routerboard-lhg-2nd", "qca,qca9533";
+	model = "MikroTik RouterBOARD LHG 2nD";
+
+	aliases {
+		led-boot = &led_user;
+		led-failsafe = &led_user;
+		led-upgrade = &led_user;
+		led-running = &led_user;
+		serial0 = &uart;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,115200n8";
+	};
```

Comment:
Just pushed a patch to master. chosen node can be dropped here.

Reply:
Done, thanks Adrian.